CITATION.cff

```diff
@@ -0,0 +1,23 @@
+# Parser settings.
+cff-version: 1.2.0
+message: Please cite this crate using these information.
+
+# Version information.
+date-released: 2022-11-12
+version: 0.4.23
+
+# Project information.
+abstract: Date and time library for Rust
+authors:
+  - alias: .
+    # email:
+    family-names: .
+    given-names: .
```

Comment:
The given and family names describe the name of *one* person.  Each person to be named receives one author entry with all of their information they would like to be named.  There is an [ongoing discussion](https://github.com/citation-file-format/citation-file-format/issues/329) on how to deal with certain regional naming conventions but the current version's practice to distinct into given and family names should work for most purposes.